network/load-balancer/how-to/create-manage-flex-ips.mdx

```diff
@@ -0,0 +1,95 @@
+
+- You can create and hold flexible IP addresses independently of any Load Balancer.
+- When you create a Load Balancer with public accessibility, you can choose whether to create a new flexible IP address with it, or use an existing flexible IP address you hold.
+- When you delete the Load Balancer, you can choose to also delete the flexible IP address, or else hold it in your account to use with a future Load Balancer you create.
+
+<Message type="note">
+The following limitations apply to Load Balancer flexible IPs:
+- You cannot currently change the flexible IP address of an existing Load Balancer, that is to say you cannot detach its flexible IP after creation to attach a different one.
+- Flexible IPs are associated with a particular product and cannot be moved to a different type of resource. For example, a flexible IP for Load Balancer can be used with another Load Balancer, but it cannot be used with other resource types like Instances or Elastic Metal servers.
+</Message>
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a Load Balancer](/network/load-balancer/how-to/create-load-balancer/)
+</Message>
+
+## How to create a flexible IP address
+
+When you create a Load Balancer, the creation wizard lets you create a new flexible IP address that will be automatically attached to the Load Balancer in question. See the [How to create a Load Balancer](/network/load-balancer/how-to/create-load-balancer/) for full details. However, if you wish to create a flexible IP address to hold independently of any Load Balancer, follow the steps below:
+
+1. Click **Load Balancers** in the **Network** section of the [Scaleway console](https://console.scaleway.com) side menu.
+
+2. Click the **Flexible IPs** tab.
+
+    <Message type="tip">
+    You must have already created a Load Balancer in order for this tab to be visible.
+    </Message>
+
+3. Click **Create flexible IP**.
+
+    The flexible IP creation wizard displays.
+
+4. Choose the **Availability Zone** you want to create your flexible IP in. 
+
+5. Check the estimated cost, and click **Create flexible IP**.
+
+    Your flexible IP address is created. You are returned to the flexible IPs tab, where your list of IPs displays.
+
+
+## How to edit the reverse DNS of a flexible IP address
+
+When you create a flexible IP address, a reverse DNS is created for it, using an auto-generated domain. You can edit the reverse DNS of any of your flexible IP addresses at any time. This allows you to replace the automatically-generated domain with your own domain name, so that, with the correct configuration of your domain's DNS, you can resolve your domain to the IP address of the flexible IP and vice versa.
```

Comment:
```suggestion
When you create a flexible IP address, a reverse DNS record is created for it, using an auto-generated domain. You can edit the reverse DNS of any of your flexible IP addresses at any time. This allows you to replace the automatically-generated domain with your own domain name, so that, with the correct configuration of your domain's DNS, you can resolve your domain to the IP address of the flexible IP and vice versa.
```